### PR TITLE
Add io.github.Rangi42.tilemap-studio

### PR DESCRIPTION
**Pull request by JakobDev**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. Rangi42/tilemap-studio#77
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

The used version of fltk doesn't support Portals, so host access is needed to work.

### CODE REVIEW

**Comment by JakobDev** [comment]
bot, build io.github.Rangi42.tilemap-studio

File: io.github.Rangi42.tilemap-studio.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+id: io.github.Rangi42.tilemap-studio
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: tilemapstudio
+finish-args:
+  - --filesystem=host:rw
```
Comment (hfiguiere) [inline io.github.Rangi42.tilemap-studio.yaml:7]: why so broad permissions?

Comment (JakobDev) [reply io.github.Rangi42.tilemap-studio.yaml:7]: If you would read the first post

> The used version of fltk doesn't support Portals, so host access is needed to work.

Comment (hfiguiere) [reply io.github.Rangi42.tilemap-studio.yaml:7]: still doesn't justify `host`.

**Pull request closed without merging**